pkg/cmd/secrets.go

```diff
@@ -520,6 +605,7 @@ func init() {
 	secretsSetCmd.Flags().StringP("project", "p", "", "project (e.g. backend)")
 	secretsSetCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
 	secretsSetCmd.Flags().Bool("raw", false, "print the raw secret value without processing variables")
+	secretsSetCmd.Flags().Bool("no-prompt", false, "do not allow entering secret value via interactive mode")
```

Comment:
I originally considered calling this `no-interactive`, but we already make use of `no-prompt` for `doppler setup`. Maybe we should switch to `no-interactive` though, since that seems to be used by other CLIs I've used. Thoughts?

Reply:
I think `no-interactive` makes sense in this use case